client/controller/index.web.js

```diff
@@ -85,23 +85,25 @@ function renderSeparateTrees( context ) {
 	renderSecondary( context );
 }
 
-function renderPrimary( context ) {
-	const { primary, store } = context;
+function renderPrimary(context, next) {
```

Comment:
Do we still need `renderPrimary`, `renderSecondary` or `renderSeparateTrees`? They seem to be obsolete with the new approach. In addition they are no longer adequate to what they do.